spotify/i18n/strings_nl.json

```diff
@@ -1,12 +1,70 @@
 {
+  "SPOTIFY":"Spotify",
+  "USERNAME":"Gebruikersnaam",
+  "PASSWORD":"Wachtwoord",
   "SPOTIFY_USERNAME":"Spotify gebruikersnaam",
   "SPOTIFY_PASSWORD":"Spotify wachtwoord",
   "HIGH_BITRATE":"Hoge kwaliteit",
+  "REFRESH_TOKEN":"Refresh Token",
   "SEARCH_RESULTS":"Aantal resultaten",
   "PLUGINS":"Last.fm",
   "LAST_FM_USERNAME":"Last.fm gebruikersnaam",
   "LAST_FM_PASSWORD":"Last.fm wachtwoord",
   "SEARCH_SONGS_SECTION":"Spotify nummers",
   "SEARCH_ALBUMS_SECTION":"Spotity albums",
   "SEARCH_ARTISTS_SECTION":"Spotify artiesten"
```

Comment:
@LinkeLoutje @volumio FYI syntax error - missing comma

PS I recommend to integrate eslint and prettier to eliminate such kind of mistakes, and many others existing in this repo 😉